helpers/user_agent_helper_test.go

```diff
@@ -0,0 +1,24 @@
+package helpers_test
+
+import (
+	"go-crawler-challenge/helpers"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("UserAgentHelper", func() {
+	Describe("#RandomUserAgent", func() {
+		It("returns a random DESKTOP browser user-agent", func() {
+			userAgent := helpers.RandomUserAgent()
+
+			Expect(len(userAgent)).NotTo(BeZero())
+		})
+
+		It("returns user-agent format", func() {
+			userAgent := helpers.RandomUserAgent()
+
+			Expect(userAgent).To(MatchRegexp(`(Firefox\/\d{2}.\d|Chrome\/\d{2}.\d.\d{4}.\d{1,3})`))
```

Comment:
@Lahphim Do you think we need to test the os part also? 🤔

Reply:
@carryall Added here https://github.com/Lahphim/go-crawler-challenge/pull/54/commits/73d8b81cb6705417a4ab6bf5911abb1628c6bc8f